### ISSUE TITLE
[Bug] You know this scroll

### ISSUE THREAD

**Issue by Emefar** [DinoDevs/GladiatusCrazyAddon#117]
I don't have this scroll. But warning says "You know this scroll"
![image](https://user-images.githubusercontent.com/36764954/48832013-375dd980-ed89-11e8-85f0-d6fb2c4c4a84.png)

**Comment by Emefar** [comment]
I have this scroll "Ölüm" not "ölümsüz". I think there is a problem with name control.
![image](https://user-images.githubusercontent.com/36764954/48832179-a1767e80-ed89-11e8-908a-25d67cfecb3d.png)

**Comment by Emefar** [comment]
Maybe you can parse scroll name:

For ex: "Scroll Death"
scroll [0] = Scroll 
scroll [1] = Death

or: Lucius Scroll
scroll [0] = Lucius
scroll [1] = Scroll

Code algorith should be like this: (Just maybe  😃)
If(scroll [0].equals(my_any_scroll) || scroll [1].equals(my_any_scroll))

**Comment by GramThanos** [comment]
Up until a month ago, the only way we could parse an item's suffix prefix was to analyze it's name. But I have decoded the item's hash dataset (which I explained on the gladiatus forum [here](https://forum.gladiatus.gameforge.com/forum/thread/1995-prefixes-suffixes-and-materials/?postID=80543#post80543)) so we should change the code to check the hash instead of the name.

I was wondering if my first implementation could be trusted 😄 but I forgot to change it to the new way.
